pkg/specgen/generate/config_linux.go

```diff
@@ -137,22 +140,29 @@ func DevicesFromPath(g *generate.Generator, devicePath string) error {
 	return addDevice(g, strings.Join(append([]string{resolvedDevicePath}, devs[1:]...), ":"))
 }
 
-func BlockAccessToKernelFilesystems(privileged, pidModeIsHost bool, g *generate.Generator) {
+func BlockAccessToKernelFilesystems(privileged, pidModeIsHost bool, mask, unmask string, g *generate.Generator) {
+	defaultMaskPaths := []string{"/proc/acpi",
```

Comment:
Can we move that outside the func on the package level? That would spare us from allocating the memory for each call.

Reply:
Please don't. What you're describing got us a CVE in Varlink land - much safer to generate a fresh copy of this for each use.